packages/trpc/server/routers/viewer/bookings/requestReschedule.handler.ts

```diff
@@ -167,6 +168,7 @@ export const requestRescheduleHandler = async ({ ctx, input }: RequestReschedule
   const builder = new CalendarEventBuilder();
   builder.init({
     title: bookingToReschedule.title,
+    bookerUrl: await getBookerUrl(user),
```

Comment:
The same way we do it in handleNewBooking, provide bookerUrl so that all links can use that as the origin.